src/cli/tests/Flow/CLI/Tests/Integration/FileSchemaCommandTest.php

```diff
@@ -90,57 +90,57 @@ public function test_run_schema_with_pretty_output() : void
     {
         "ref": "order_id",
         "type": {
-            "type": "uuid",
-            "nullable": false
+            "type": "uuid"
         },
+        "nullable": false,
         "metadata": []
     },
     {
         "ref": "created_at",
         "type": {
-            "type": "string",
-            "nullable": false
+            "type": "string"
         },
+        "nullable": false,
         "metadata": []
     },
     {
         "ref": "updated_at",
         "type": {
-            "type": "string",
-            "nullable": false
+            "type": "string"
         },
+        "nullable": false,
         "metadata": []
     },
     {
         "ref": "discount",
         "type": {
-            "type": "string",
-            "nullable": true
+            "type": "string"
```

Comment:
`string`?

Reply:
this particular test reads schema from a CSV file, without passing option `--auto-cast` to the command it will not try to guess the schema from the `shape of data` and when you read csv in php, everything is string: https://3v4l.org/7JSB5